lib/cosmos/interfaces/protocols/length_protocol.rb

```diff
@@ -135,7 +135,7 @@ def reduce_to_single_packet
       packet_length = (length * @length_bytes_per_count) + @length_value_offset
 
       # Make sure we have enough data for the packet
-      return:STOP if @data.length < packet_length
+      return :STOP if @data.length < packet_length
```

Comment:
Also fixed this little nugget. Looks like we've never hit this case yet.

Reply:
return:STOP in IRB works without the space.   Good to know!